src/Features/JsonPatch.SystemTextJson/src/JsonPatchDocument.cs

```diff
@@ -218,4 +221,19 @@ IList<Operation> IJsonPatchDocument.GetOperations()
 
         return allOps;
     }
+
+    /// <summary>
+    /// Populates metadata for the related endpoint when this type is used as a parameter.
+    /// </summary>
+    /// <param name="parameter">The <see cref="ParameterInfo"/> for the endpoint parameter.</param>
+    /// <param name="builder">The endpoint builder for the endpoint being constructed.</param>
+#pragma warning disable RS0016 // Add public types and members to the declared API
```

Comment:
Don't use #pragma warning disable here. Instead, update the `PublicApi.Unshipped.txt` file in this directory with these APIs.